### PR TITLE
Docstring fix for curriculum learning callback

### PR DESCRIPTION
**Pull request by snarayan21**
Docstring was slightly wrong for curriculum learning callback, this is a quick fix.

### CODE REVIEW

File: llmfoundry/callbacks/curriculum_learning_callback.py [hunk 1]
```diff
@@ -28,9 +28,9 @@ class CurriculumLearning(CallbackWithConfig):
     """Starts an epoch with a different dataset when resuming from a checkpoint.
 
     Args:
-        dataset_index (int): The index of the dataset currently being used.
-        current_dataset_config (Dict): The configuration of the dataset currently
+        train_config (Dict): The configuration of the dataset currently
```
Comment (dakinggg) [inline llmfoundry/callbacks/curriculum_learning_callback.py:31]: Note that this is the full train config and must contain a `train_loader` key

**Review by dakinggg — APPROVED** [review]

File: llmfoundry/callbacks/curriculum_learning_callback.py (hunk 1 above)
Comment (snarayan21) [reply llmfoundry/callbacks/curriculum_learning_callback.py:31]: do you want me to note that in the docstring? or just noting here

Comment (dakinggg) [reply llmfoundry/callbacks/curriculum_learning_callback.py:31]: in the docstring plz

Comment (snarayan21) [reply llmfoundry/callbacks/curriculum_learning_callback.py:31]: done

**Pull request merged**